### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by whoburg**
@bqpd, this is ready to review pending Jenkins. Addresses comments in #454

### CODE REVIEW

File: gpkit/_mosek/cli_expopt.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def imize(c, A, p_idxs, *args, **kwargs):
         """
 
         with open(filename, "w") as f:
-            numcon = 1+p_idxs[-1]
+            numcon = p_idxs[-1]
```
Comment (bqpd) [inline gpkit/_mosek/cli_expopt.py]: Ah, great.

File: gpkit/model.py [hunk 2]
```diff
@@ -252,7 +252,7 @@ def allsubs(self):
     def signomials_et_al(self):
         "Get signomials, beforesubs, allsubs in one pass; applies VarKey subs."
         signomials = self.signomials
-        beforesubs = NomialData.fromnomials(signomials)
+        beforesubs = self.beforesubs
```
Comment (bqpd) [inline gpkit/model.py]: Err, no, this is slower.

Comment (whoburg) [inline gpkit/model.py]: really? how? aren't they equivalent?

Comment (bqpd) [inline gpkit/model.py]: Nope. 

```python
    @property
    def beforesubs(self):
        "Get this Model's NomialData before any substitutuions"
        return NomialData.fromnomials(self.signomials)
```

Comment (whoburg) [inline gpkit/model.py]: ah, ok. I'll update the PR  -- standby.

**Comment by whoburg** [comment]
ready for re-review

**Comment by bqpd** [comment]
Looks good!

**Comment by whoburg** [comment]
Shoot, I think I broke master. @galbramc the CI on master reports "aborted"

**Comment by galbramc** [comment]
no I am changing things on jenkins. Because of the 'globa' $HOME/.gpkit folder you cannot have multiple tests running at the same time on any given machine. I realized that I need to make sure the gpkit_Push and gpkit_PullRequests could not be running at the same time. I found a plugin that was even better. It allows me to restrict things so no two jobs can run at the same time on a given machine, but jobs can run in parallel across machines. Before your builds were running in serial across all machines, but now you are parallel across machines and serial on a given machine.

**Comment by whoburg** [comment]
awesome!!

**Comment by bqpd** [comment]
Nifty! (looking forward to removing $HOME/.gpkit)

**Comment by galbramc** [comment]
Yes. If you remove $HOME/.gpkit so that each build is completely independent, then we can run all builds completely in parallel. Should you open an issue maybe about this?

**Comment by whoburg** [comment]
I believe that is captured in #315 .

**Comment by galbramc** [comment]
Yes it is.

**Comment by galbramc** [comment]
Actually, I think you can check off that first goal. You are no longer mucking with a users .bashrc or .bash_profile, so no need to ask for permission.

**Pull request merged**